packages/amplication-client/src/Application/git/AppGitStatusPanel.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { Button, EnumButtonStyle } from "../../Components/Button";
+import { Icon, Label, Tooltip } from "@amplication/design-system";
+import { Link } from "react-router-dom";
+import * as models from "../../models";
+import { isEmpty } from "lodash";
```

Comment:
it's better to import the specific lodash function then the library. e.g. import isEmpty from 'lodash.isempty'